### ISSUE TITLE
Abilitiy to add theshold date or due date in `ice_recur.txt` file for recurring events

### ISSUE THREAD

**Issue by JSpenced** [rlpowell/todo-text-stuff#17]
Currently, it doesn't make sense to add a threshold date or due date because these are always changing. I use threshold dates frequently. Ideally, I would put `t:today` where `today` is the current date for every entry added into todo.txt from `ice_recur`.

For example this entry:
`weekly ; day Sunday - Weekly review +project @context`

you could do maybe something like this:
`weekly ; day Sunday - Weekly review +project @context - today`
or maybe tap into the current system so could write:
`weekly ; day Sunday - Weekly review +project @context - weekly ; day Sunday`

Then this would be translated into:
`weekly ; day Sunday - Weekly review +project @context t:2018-12-05`

Ideally, you could do this for due date and threshold date. But for me, I prefer being able to set the threshold date as I often want to add it set to the current day. Is this possible?

**Comment by rlpowell** [comment]
Is there any reason not to just replace "t:today"?

**Comment by JSpenced** [comment]
I guess it's because the program I use highlights (in blue) and treats `t:` as a threshold (or due date) only if it is in this exact date format string: `t:2018-12-14`. Then it can easily be changed via key bindings. If it's written as `t:today` then it treats it as a text tag not a threshold date. 

Anyways, I just put in now `t:0000-00-00` so it treats it as a threshold as a placeholder and it's fine. It was just a feature I had thought about but not really necessary.

**Comment by rlpowell** [comment]
Erm, no, that's not what I meant.  I meant what if ice_recur just replaced "t:today" with "t:[the-actual-date-string]"?

**Comment by JSpenced** [comment]
Yeah, that could be good. Is it possible to make it so you can put commands like "t:today+10d" or "t:today+1w" or "t:today+1m". For threshold maybe this doesn't make as much sense but it could for due date so could right "due:today+3d" or "due:today+1w".

Anyways, yeah `t:today` would help a lot even if none of the above was included.
